### ISSUE TITLE
OPKG set http_proxy doesn't work. It's very important in China

### ISSUE THREAD

**Issue by fannwong001** [openwrt/openwrt#12252]
### Describe the bug

in file /etc/opkg.conf add :
option http_proxy http://192.168.1.3:1080

execute: 
opkg update

Then 192.168.1.3 does not receive any packet from the host. And the download speed is very slow in China.
And if set https_proxy , it will return error.

### OpenWrt version

r20068-788a0cf15c

### OpenWrt target/subtarget

x86/64

### Device

VMware, Inc. VMware Virtual Platform

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for OpenWrt, not an unsupported fork.

**Comment by brada4** [comment]
Slash in the end? 1080 is socks typically, not http(S)?

**Comment by fannwong001** [comment]
> Slash in the end? 1080 is socks typically, not http(S)?

that is definitely http. which for http or socks is manually set.

**Comment by brada4** [comment]
Set slash in end of setting like `http://123.456.789.0:1234/`

**Comment by fannwong001** [comment]
> Set slash in end of setting like `http://123.456.789.0:1234/`

Now I set 
option https_proxy http://192.168.1.3:1080
or before execute command"opkg update", set environment https_proxy, then It works.
It's because of opkg use wget to download.And wget will use https_proxy to get url with https, not like curl.
And I said option https_proxy once a time returned error just because I set option https_proxy after option http_proxy, and this happened in an old version.
